api/lib/round-tracker.js

```diff
@@ -264,10 +268,11 @@ export async function maybeCreateSparkRound (pgClient, {
       $2,
       $3,
       $4,
-      (
-        $5::int /* previousRound.max_tasks_per_node || BASELINE_TASKS_PER_NODE */
-        * $6::int /* TASKS_EXECUTED_PER_ROUND */
-        / $7::int /* previousRound.measurement_count || TASKS_EXECUTED_PER_ROUND */
+      LEAST(
+        $5,
```

Comment:
Please add a comment to explain that `$5` is for `MAX_TASKS_PER_NODE_LIMIT`

Reply:
https://github.com/filecoin-station/spark-api/commit/58dc884f9f2295edc62ee4ee39beee0bdc6f0ce0